LoopKit/DosingDecisionStore.swift

```diff
@@ -226,6 +226,36 @@ extension DosingDecisionStore {
             }
         }
     }
+    
+    public func findDosingDecisionsByDate(_ date: Date) async throws -> [StoredDosingDecision] {
```

Comment:
Maybe some documentation that this isn't fetching a dosing decision that matches the specific date, but a range (and how big the range is).

Reply:
@ps2 - Okay, I've found a way to  query this by the id instead; just required an `id` CoreData column for `DosingDecisionObject` that mirrors the id of the data blob.